web/tests/integration/components/related-resources/add-test.ts

```diff
@@ -301,4 +301,4 @@ module("Integration | Component | related-resources/add", function (hooks) {
       .dom("[data-test-related-resources-search-loading-icon]")
       .exists("a loading icon is shown when `searchIsRunning` is true");
   });
-});
+});
```

Comment:
😮 Was this accidental or does ESLint have something against EOF in files?

Reply:
Accidental. In trying to quickly revert a change, I copied and pasted from GitHub, which apparently cuts off the whitespace.